### PR TITLE
Provide script check for icons

### PR DESCRIPTION
**Pull request by mkondratek**

### CODE REVIEW

File: scripts/validate-icons [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+ICONS_PATH='frontend/icons/src/main/resources'
+icon_files=$(git ls-files "$ICONS_PATH/*.svg")
+declared_icons=$(git grep -Pho '(?<=Icon) .* (?=\= loadIcon)' -- 'frontend/icons/src/main/java/icons/MacheteIcons.java')
+
+for referenced_file in $(git grep -Pho '(?<=loadIcon\("/).*(?="\))' -- 'frontend/icons/src/main/java/icons/MacheteIcons.java'); do
+  referenced_file_path="$ICONS_PATH/$referenced_file"
+  referenced_file_path_dark=${referenced_file_path::-4}_dark.svg
```
Comment (PawelLipski) [inline scripts/validate-icons:14]:
```suggestion
  referenced_file_path_dark=${referenced_file_path%.svg}_dark.svg
```

File: scripts/validate-icons [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  if [[ ! $icon_files =~ $referenced_file_path ]]; then
```
Comment (PawelLipski) [inline scripts/validate-icons:15]: `=~` is for regexes... can't it be a regular `!=` here?

File: scripts/validate-icons [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    die "Icon '$referenced_file' referenced from MacheteIcons.java is missing from resources!"
+  fi
+  if [[ ! $icon_files =~ $referenced_file_path_dark ]]; then
+    die "Dark icon '$referenced_file' referenced from MacheteIcons.java is missing from resources!"
```
Comment (PawelLipski) [inline scripts/validate-icons:19]: Actually... all these changes to svgs are resizes? 🤔

File: scripts/validate-icons (hunk 2 above)
Comment (mkondratek) [reply scripts/validate-icons:15]: does `!=` work like contains?

File: scripts/validate-icons (hunk 3 above)
Comment (mkondratek) [reply scripts/validate-icons:19]: changes to svgs are mostly reformatiings (and some cleanup) - no resizes

File: scripts/validate-icons (hunk 2 above)
Comment (PawelLipski) [reply scripts/validate-icons:15]: Oh!

Then some `*...*` could be useful

Comment (mkondratek) [reply scripts/validate-icons:15]: whhat?

Comment (PawelLipski) [reply scripts/validate-icons:15]:
```suggestion
  if [[ $icon_files != "*$referenced_file_path*" ]]; then
```

File: scripts/validate-icons (hunk 3 above)
Comment (PawelLipski) [reply scripts/validate-icons:19]: Oh, xml reformattings 🤦🏻 I keep forgetting that svg is xml

**Review by PawelLipski — APPROVED** [review]

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**